src/ResourceGraph/ResourceGraph/ResourceGraph.csproj

```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="0.10.0-preview" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Accounts\Accounts\Accounts.csproj" />
```

Comment:
@OlhaTkachenko this reference should be removed

Reply:
@cormacpayne , thank you for updating ARM client! I removed this reference.